packages/next-international/src/app/client/create-use-change-locale.ts

```diff
@@ -31,6 +31,11 @@ export function createUseChangeLocale<LocalesKeys>(
     }
 
     return function changeLocale(newLocale: LocalesKeys) {
+      if (newLocale === currentLocale) {
+        warn(`The locale '${newLocale}' is already set`);
```

Comment:
I don't think we should actually produce any log when this happens. Just returning early should be fine.

Reply:
yap, done